src/util.ts

```diff
@@ -109,8 +109,9 @@ export async function fetchFrom(
   url: string,
   options?: http.RequestOptions
 ): Promise<string> {
+  const requestOptions: http.RequestOptions = { ...options, timeout: 5000 };
```

Comment:
[nit]
```suggestion
  const requestOptions: http.RequestOptions = { timeout: 5000, ...options };
```

so we can specify the timeout millis from callee of `fetchFrom`

Reply:
oh yeah, I forgot that order matters